integration-tests/tests/src/tests/sync/user.ts

```diff
@@ -437,90 +436,4 @@ describe.skipIf(environment.missingServer, "User", () => {
       await expect(user.functions.error()).to.be.rejectedWith("function not found: 'error'");
     });
   });
-
-  describe("mongo client", () => {
```

Comment:
Similar tests exist in [mongo-db-client.ts](https://github.com/realm/realm-js/blob/main/integration-tests/tests/src/tests/sync/mongo-db-client.ts).